### PR TITLE
33/social impact beyond business section

### PR DESCRIPTION
**Pull request by NewtonOmbese**
# Description

Created a section that describes the Social Impact beyond business section that shows an image on the right side, a paragraph of text and a title. This component displays provided text and images side by side on the social impact section using the reusable animated button components in issue https://github.com/italanta/elewa-group/issues/31. 

To achieve this, I needed to:
- Step 1: Create the Social Impact beyond business section component
```
nx g @nrwl/angular:component pages/elewa/social-impact/social-impact-beyond-business-section
```
- Make an import of the button module to be reused in the component

Fixes # (issue  https://github.com/italanta/elewa-group/issues/33) 

## Type of change
- [ ] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
Desktop View
updated

Mobile View
beyond-business-mobile

# How Has This Been Tested?
Step 1: Added the component to social-impact-page.component.html
Step 2: Started the server by running ```nx serve elewa-group-website```
Step 3: Using chrome developer tools to test on the responsiveness of the website

# Checklist:t
- [ ] I have performed a self-review of my code
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by NewtonOmbese** [comment]
Review and merge @IanOdhiambo9

**Comment by NewtonOmbese** [comment]
Updated the Paragraphs as they were declared as arrays
Review and Merge @IanOdhiambo9

File: libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts [hunk 1]
```diff
@@ -4,10 +4,17 @@ import { LayoutModule } from '@elewa-group/elements/layout';
 import { SocialImpactPageComponent } from './pages/social-impact-page/social-impact-page.component';
 import { SocialImpactHeroSectionComponent } from './components/social-impact-hero-section/social-impact-hero-section.component';
 import { SocialImpactRoutingModule } from './social-impact.routing';
+import { SocialImpactBeyondBusinessSectionComponent } from './components/social-impact-beyond-business-section/social-impact-beyond-business-section.component';
+
+import { BannersModule } from '@elewa-group/features/components/banners';
```
Comment (LemmyMwaura) [inline libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts:9]: Fix the order of imports

Comment (LemmyMwaura) [inline libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts:9]: Fix the order of imports

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Fix the imports

**Comment by NewtonOmbese** [comment]
@LemmyMwaura done fixing the order

**Comment by NewtonOmbese** [comment]
@LemmyMwaura  new issue fix review and give feedback

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Looks good, Sync the latest updates

**Comment by NewtonOmbese** [comment]
Done syncing

**Comment by IanOdhiambo9** [comment]
Closing this because newton can't work on one points anymore, but good job!

**Pull request closed without merging**